DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateGroups/GroupDelegatesRemove.cshtml

```diff
@@ -0,0 +1,89 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateGroups
+@using Microsoft.Extensions.Configuration
+@model GroupDelegatesRemoveViewModel
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = errorHasOccurred ? "Error: Remove delegate from group" : "Remove delegate from group";
+  ViewData["Application"] = "Tracking System";
+  ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/TrackingSystem/Centre/Dashboard";
+  ViewData["HeaderPathName"] = "Tracking System";
+  var confirmError = ViewData.ModelState[nameof(GroupDelegatesRemoveViewModel.Confirm)]?.Errors?.Count > 0;
+  var confirmFormErrorClass = confirmError ? "nhsuk-form-group--error" : "";
+}
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    @if (errorHasOccurred) {
+      <vc:error-summary order-of-property-names="@new []{ nameof(GroupDelegatesRemoveViewModel.Confirm) }" />
+    }
+
+    <h1 class="nhsuk-heading-xl">Are you sure you would like to remove @Model.DelegateName from this group?</h1>
+  </div>
+</div>
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-one-quarter nhsuk-heading-l">
+    <div class="nhsuk-u-font-weight-bold">
+      Group name:
+    </div>

```

Comment:
Do we need this additional div here or could we specify the bold font on the parent?